### PR TITLE
PLT-2406 Add "Database:" to Mattermost About Dialog

### PR DESCRIPTION
**Pull request by tbalthazar**
The property has been named SQLDriverName to make a difference with the
FileSettings DriverName.

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: webapp/components/about_build_modal.jsx [hunk 1]
```diff
@@ -137,6 +137,13 @@ export default class AboutBuildModal extends React.Component {
                                     />
                                     {'\u00a0' + config.Version + '\u00a0' + config.BuildNumber}
                                 </div>
+                                <div>
+                                    <FormattedMessage
+                                        id='about.database'
```
Comment (enahum) [inline webapp/components/about_build_modal.jsx]: is this a new string?

if it is please add it to `webapp/i18n/en.json`.

Thanks

**Comment by tbalthazar** [comment]
@enahum Done! :)

**Comment by enahum** [comment]
Thanks @tbalthazar.

+1

**Pull request merged**